tests/unit_tests/tags/commands/update_test.py

```diff
@@ -30,7 +31,7 @@ def session_with_data(session: Session):
     from superset.models.sql_lab import Query, SavedQuery
     from superset.tags.models import Tag
 
-    engine = session.get_bind()
+    engine = db.session.get_bind()
```

Comment:
Same comment as before about enhancing the session with data.

Reply:
See https://github.com/apache/superset/pull/26909#issuecomment-1922328534.